app/views/dashboard/index.html.erb

```diff
@@ -1,6 +1,8 @@
 <h1>Dashboard</h1>
 
-<ul>
-  <li>Name: <%= @user.name %></li>
-  <li>Username: <%= @user.username %></li>
-</ul>
+<% if current_user %>
```

Comment:
Do we want to allow a visitor to hit this page? Or should we put a before filter on the dashboard controller and redirect them if they aren't logged in?

Reply:
I think that we should allow a visitor to access this page for now and adjust it as neccessary depending on future user stories.